src/config/multer.js

```diff
@@ -0,0 +1,15 @@
+const multer = require("multer");
+const ValidationReasons = require("../api/middleware/validators/validationReasons");
+
+const storage =  multer.memoryStorage();
+const limits = { fileSize: 5000000 };
+const fileFilter = (req, file, cb) => {
+    const fileTypes = /png|jpeg|jpg/;
+    const mimeType = fileTypes.test(file.mimetype);
+    if (mimeType) {
+        cb(null, true);
+    } else {
+        cb(new Error(ValidationReasons.IMAGE_FORMAT));
```

Comment:
Might need to change this into an APIError instance, after #117 to make it appear on the response, otherwise a generic error will be shown instead

Reply:
This error is caught in the function `parseSingleFile`; therefore, it doesn't need to change